web-app/src/InvoiceDocumentation/ubl2.1/rules/Rule.js

```diff
@@ -2,6 +2,16 @@ import {useSelector} from 'react-redux'
 import {useTranslation} from 'react-i18next'
 import NotFound from '../../../helpers/NotFound'
 import {ubl21RuleSelector} from '../../../cache/documentation/state'
+import {Link} from 'react-router-dom'
```

Comment:
let's have absolute imports first and relative imports later, so move it to line `2`